src/encoded/types/experiment.py

```diff
@@ -177,64 +177,28 @@ def biosample_summary(self,
-                                                       request.embed(target_name, '@@object'),
-                                                       None)
-                                    construct_objects_list.append(item_to_add)
-
-                            model_construct_objects_list = None
-                            model_organism_donor_constructs = biosampleObject.get(
-                                'model_organism_donor_constructs')
-                            if model_organism_donor_constructs is not None and \
-                               len(model_organism_donor_constructs) > 0:
-                                model_construct_objects_list = []
-                                for c in model_organism_donor_constructs:
-                                    construct_object = request.embed(c, '@@object')
-                                    target_name = construct_object['target']
-                                    if 'promoter_used' in construct_object and \
-                                       construct_object['promoter_used'] is not None:
-                                        promo = construct_object['promoter_used']
-                                        item_to_add = (construct_object,
-                                                       request.embed(target_name, '@@object'),
-                                                       request.embed(promo, '@@object'))
-                                    else:
-                                        item_to_add = (construct_object,
-                                                       request.embed(target_name, '@@object'),
-                                                       None)
-                                    model_construct_objects_list.append(item_to_add)
-
-                            rnai_objects = None
-                            rnais = biosampleObject.get('rnais')
-                            if rnais is not None and len(rnais) > 0:
-                                rnai_objects = []
-                                for r in rnais:
-                                    rnai_object = request.embed(r, '@@object')
-                                    target_object = request.embed(rnai_object['target'], '@@object')
-                                    rnai_info = {'rnai_type': rnai_object['rnai_type'],
-                                                 'target': target_object['label']}
-                                    rnai_objects.append(rnai_info)
+                                    modification_dict = {'category': gm_object.get('category')}
+                                    if gm_object.get('modified_site_by_target_id'):
+                                        modification_dict['target'] = request.embed(
+                                            gm_object.get('modified_site_by_target_id'),
+                                                          '@@object')['label']
```

Comment:
This is an interesting case where we embed ALMOST the whole object but stop at Target... then have to go GET it anyway.

Reply:
are you proposing to embed the target?

Reply:
I guess I am suggesting it as a possibility?  Not sure if it would be better or worse.